pyEpiabm/pyEpiabm/tests/test_unit/test_core/test_person.py

```diff
@@ -79,6 +85,11 @@ def test_configure_place(self):
         self.assertEqual(len(self.person.places), 0)
         self.assertRaises(KeyError, self.person.remove_place, test_place_2)
 
+    def test_vaccinate(self):
+        self.person.vaccinate(5)
```

Comment:
```suggestion
        self.person.vaccinate(time=5)
```
Providing params with keywords like this is an easy way to make code more readable